tests/cypress/support/commands.js

```diff
@@ -359,3 +359,103 @@ Cypress.Commands.add('logoutFromPublisher', () => {
     cy.get('#usernameUserInput').should('exist');
 })
 
+Cypress.Commands.add('publishThirdPartyApi', (apiName = null) => {
+
+    cy.visit(`${Utils.getAppOrigin()}/publisher/apis`);
+
+    //check if other protocol option is added in AsyncApi
+    cy.get('#itest-streaming-api-create-menu', { timeout: 30000 });
+    cy.get('#itest-streaming-api-create-menu').click();
+    cy.get('#itest-id-create-streaming-api-import').click();
+    cy.get('#outlined-full-width').type('https://raw.githubusercontent.com/asyncapi/spec/v2.0.0/examples/2.0.0/streetlights.yml');
+    cy.get('#outlined-full-width').should('have.value','https://raw.githubusercontent.com/asyncapi/spec/v2.0.0/examples/2.0.0/streetlights.yml');
```

Comment:
Line length should be 120 chars maximum.

Reply:
This is fixed in [#314](https://github.com/wso2/apim-apps/pull/314)